llvm/lib/IR/ConstantRangeList.cpp

```diff
@@ -81,6 +81,86 @@ void ConstantRangeList::insert(const ConstantRange &NewRange) {
   }
 }
 
+ConstantRangeList
+ConstantRangeList::unionWith(const ConstantRangeList &CRL) const {
+  assert(getBitWidth() == CRL.getBitWidth() &&
+         "ConstantRangeList types don't agree!");
```

Comment:
nit: maybe "bitwidths" instead of "types" to be more specific?

Reply:
Done.